roles/configure_ztp_gitops_apps/tasks/main.yaml

```diff
@@ -103,30 +124,40 @@
         - "  - group: hive.openshift.io"
 
     - name: Add ClusterImageSet to the policies-app-project.yaml
-      lineinfile:
+      ansible.builtin.lineinfile:
         path: "{{ temp_dir.path }}/ztp/argocd/deployment/policies-app-project.yaml"
         line: "{{ item }}"
         insertafter: clusterResourceWhitelist.*
       loop:
         - "    kind: ClusterImageSet"
         - "  - group: hive.openshift.io"
 
-    - name: Run the policies and cluster apps
-      shell: |
-        {{ czga_oc_tool_path }} --kubeconfig="{{ temp_dir.path }}/kubeconfig" apply -k "{{ temp_dir.path }}/ztp/argocd/deployment"
-
     - name: Replace image container URL in argocd-openshift-gitops-patch.json
-      replace:
+      ansible.builtin.replace:
         path: "{{ temp_dir.path }}/ztp/argocd/deployment/argocd-openshift-gitops-patch.json"
         regexp: "{{ ztp_site_generator_image }}.*/"
         replace: "{{ ztp_site_generator_image }}:{{ ztp_site_generator_version }}"
 
-    - name: Apply ArgoCD changes
-      shell: |
+    - name: Patch the ArgoCD instance to install the GitOps ZTP plugin
+      ansible.builtin.shell: |
         {{ czga_oc_tool_path }} --kubeconfig="{{ temp_dir.path }}/kubeconfig" patch argocd openshift-gitops -n openshift-gitops --patch-file "{{ temp_dir.path }}/ztp/argocd/deployment/argocd-openshift-gitops-patch.json" --type=merge
 
+    - name: Disable proxy addon
+      community.kubernetes.k8s:
+        api_version: multicluster.openshift.io/v1
+        kubeconfig: "{{ czga_kubeconfig_path }}"
```

Comment:
If you've copied this in `{{ temp_dir.path }}/kubeconfig`, why not using this here too, like in the other commands that are running in this playbook?